src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
 // write code here
+const getList = document.querySelectorAll('li');
+const arr = Array.from(getList);
+
+function sortList(list) {
+  const sorted = list.sort((a, b) => {
+    const salaryA = parseFloat(
+      a.getAttribute('data-salary').replace(/[^0-9.-]+/g, ''),
+    );
+    const salaryB = parseFloat(
+      b.getAttribute('data-salary').replace(/[^0-9.-]+/g, ''),
+    );
+
+    return salaryA - salaryB;
+  });
+  const removeUl = document.querySelector('ul');
```

Comment:
Ensure that the `ul` element exists in the DOM before attempting to manipulate it. You might want to add a check to see if `removeUl` is not null to avoid potential runtime errors.